plugins/plugin-urlfactory/src/main/java/org/eclipse/che/plugin/urlfactory/URLFetcher.java

```diff
@@ -40,6 +42,9 @@
   /** Maximum size of allowed data. (30KB) */
   protected static final long MAXIMUM_READ_BYTES = 30 * 1000;
 
+  /** The Compiled REGEX PATTERN that can be used for http|https git urls */
+  final Pattern GIT_HTTP_URL_PATTERN = Pattern.compile("(^http[s]?://.*)(.git$)");
```

Comment:
do we need to check ssh urls here?

Reply:
am not sure Eclipse factory accepts ssh urls, thats why I thought to check only http(s)

Reply:
I don't see a reason why not.

Reply:
Based on the official doc https://www.eclipse.org/che/docs/factory/creating/index.html I don’t see a case for ssh/git protocols. Hence set the pattern currently to http/https.